src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -4792,7 +5027,7 @@ def add_github_actions(cmd, resource_group, name, repo, runtime=None, token=None
     else:
         file_name = "{}_{}.yml".format(branch.replace('/', '-'), name.lower())
     dir_path = "{}/{}".format('.github', 'workflows')
-    file_path = "/{}/{}".format(dir_path, file_name)
```

Comment:
Should we also remove the slash in `remove_github_actions` function

Reply:
leaving the other one as is per Sisira's comment on keeping this PR focused on stacks